### PR TITLE
Milestone: Mobile menu feature

### PR DESCRIPTION
**Pull request by jenadiusnicholaus**
This milestone added On click event  to display the Mobile Menu and 

1. When the user clicks (or taps) the close (X) button, the mobile menu disappears.
2. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
3. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,22 +13,31 @@
...
         <nav>
-            <div class="brand">
-              <a href="#" class="logo">My Logo</a>
-            </div>
             <div class="hamburger">
                 <ul class="menu">
                     <li class="menu-item"><a class="menu-link" href="#">Portifolio</a></li>
                     <li class="menu-item"><a class="menu-link" href="#">About</a> </li>
                     <li class="menu-item"><a class="menu-link" href="#">Contact</a></li>
                     <li class="menu-item"><a class="menu-link" href="#"><i class="fa-solid fa-envelope"></i></a></li>
```
Comment (BenMukebo) [inline index.html:25]: - [ ] Please set the correct path to each of your nav items; for example, when the user clicks on the about nav link, it should take him/her to the correct about section, please fix that.

File: assets/css/smallscreen.css [hunk 2]
```diff
@@ -1,55 +1,128 @@
...
+
+.mobile-icon-menu {
+  display: none;
+  position: absolute;
+  background-color: #3c3a39;
+  top: 40px;
+  left: 0;
+  height: calc(95vh - 50px);
+  width: 100%;
+}
```
Comment (BenMukebo) [inline assets/css/smallscreen.css]: - [ ] Kindly play with your top property to adjust this spacing above the hamburger menu.
- [ ] Consider using the same font and colors as given in the design, please take a look back to the Figma design and rectify your design with correct data 😄 
- [ YOU] 
font


- [ FIGMA]

menu
`

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@jenadius,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] Kindly, Add a descriptive README file, it should contain technologies used to build the project, as a reference please use this [template](https://github.com/microverseinc/readme-template) You can copy past but remember to customize it to your project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @jenadiusnicholaus, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] I noticed we have just a simple typo in `portfolio` in our menu, other than that perfect work 😄 
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**